### PR TITLE
docs(ShadowContainer): Adjusting docs after review

### PR DESCRIPTION
**Pull request by eriklimakc**

### CODE REVIEW

File: doc/controls/ShadowContainer.md [hunk 1]
```diff
@@ -49,8 +49,25 @@ Opacity | double | The opacity of the shadow.
 BlurRadius | double | The radius of the blur that will be applied to the shadow **[0..100]**.
 Spread | double | The spread will inflate or deflate (if negative) the control shadow size **before** applying the blur.
 
+As mentioned `IsInner` is similar to `inset` of `box-shadow` in CSS. See below an image that illustrate its usage:
+
+![CSS box-shadow without inset](../assets/box-shadow.png)
```
Comment (mrlacey) [inline doc/controls/ShadowContainer.md:54]: It looks like this and the next images are the wrong way around.
The text immediately above talks about IsInner/inset.

File: doc/controls/ShadowContainer.md [hunk 2]
```diff
@@ -49,8 +49,25 @@ Opacity | double | The opacity of the shadow.
...
+
+Without `inset` a Drop (outer) Shadow is rendered.
+
+![CSS box-shadow with inset](../assets/box-shadow-inset.png)
+
+With `inset` an Inner Shadow is rendered.
+
+For more info of how `inset` works in CSS for `box-shadow` follow [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#values).
```
Comment (mrlacey) [inline doc/controls/ShadowContainer.md:62]: "this link" is bad for usability and SEO.
Use the URL as the text for the link.

File: doc/controls/ShadowContainer.md [hunk 3]
```diff
@@ -49,8 +49,25 @@ Opacity | double | The opacity of the shadow.
...
+
+> [!IMPORTANT]
+> Be careful not to confuse the `Shadow` property with the `Shadows` property. The singular `Shadow` property comes from `UIElement`. To add shadows to the `ShadowContainer`, use only the `Shadows` property.
+
 ## Usage
 
+Observe shadows usage with the following code snippets and its results below.
```
Comment (mrlacey) [inline doc/controls/ShadowContainer.md:69]: General documentation guidance is to write naturally as if speaking to a real person. This does not sound like anyone talks. - apologies if you do talk like this.

File: doc/controls/ShadowContainer.md [hunk 4]
```diff
@@ -244,51 +262,120 @@ xmlns:utu="using:Uno.Toolkit.UI"
...
+		</utu:ShadowContainer>
 	</StackPanel>
 </StackPanel>
 ```
 
 ![neumorphism built with 2 shadows](../assets/shadows-neumorphism.png)
+
+## Theme Switching support
+
+To ensure that the Shadows feature seamlessly integrates with Theme Switching, it's essential to include each shadow definition within the Theme Dictionary. By doing so, you'll enable consistent shadow rendering across different themes.
```
Comment (jeromelaban) [inline doc/controls/ShadowContainer.md:316]:
```suggestion
To ensure that the Shadows feature seamlessly integrates with [Theme Switching](xref:Uno.Features.Themes
), it's essential to include each shadow definition within the Theme Dictionary. By doing so, you'll enable consistent shadow rendering across different themes.
```

File: doc/controls/ShadowContainer.md [hunk 5]
```diff
@@ -244,51 +262,120 @@ xmlns:utu="using:Uno.Toolkit.UI"
...
+							Opacity="1" 
+							Spread="0" 
+							Color="{StaticResource ShadeShadowColor}" />
+			</utu:ShadowCollection>
+		</ResourceDictionary>
+		<ResourceDictionary x:Key="Light">
+			<Color x:Key="LightShadowColor">Orange</Color>
+			<Color x:Key="ShadeShadowColor">Green</Color>
+
+			<utu:ShadowCollection x:Key="ButttonShadows">
```
Comment (jeromelaban) [inline doc/controls/ShadowContainer.md:351]:
```suggestion
			<utu:ShadowCollection x:Key="ButtonShadows">
```

File: doc/controls/ShadowContainer.md [hunk 6]
```diff
@@ -244,51 +262,120 @@ xmlns:utu="using:Uno.Toolkit.UI"
...
+							Opacity="1" 
+							Spread="0" 
+							Color="{StaticResource ShadeShadowColor}" />
+			</utu:ShadowCollection>
+		</ResourceDictionary>
+	</ResourceDictionary.ThemeDictionaries>
+</ResourceDictionary>
+...
+
+<utu:ShadowContainer Shadows="{StaticResource ButttonShadows}">
```
Comment (jeromelaban) [inline doc/controls/ShadowContainer.md:372]:
```suggestion
<utu:ShadowContainer Shadows="{StaticResource ButtonShadows}">
```

File: doc/controls/ShadowContainer.md [hunk 7]
```diff
@@ -49,8 +49,25 @@ Opacity | double | The opacity of the shadow.
...
+![CSS box-shadow without inset](../assets/box-shadow.png)
+
+For more info on how `inset` works in CSS for `box-shadow` follow https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#values.
+
+> [!IMPORTANT]
+> Be careful not to confuse the `Shadow` property with the `Shadows` property. The singular `Shadow` property comes from `UIElement`. To add shadows to the `ShadowContainer`, use only the `Shadows` property.
+
 ## Usage
 
+See how to add shadows to your controls with the following code and the result.
```
Comment (jeromelaban) [inline doc/controls/ShadowContainer.md:69]: We musy explain that a new package reference is needed to get shadows.

File: doc/controls/ShadowContainer.md [hunk 8]
```diff
@@ -49,8 +52,25 @@ Opacity | double | The opacity of the shadow.
 BlurRadius | double | The radius of the blur that will be applied to the shadow **[0..100]**.
 Spread | double | The spread will inflate or deflate (if negative) the control shadow size **before** applying the blur.
 
+As mentioned `IsInner` is similar to `inset` of `box-shadow` in CSS. See below an image that illustrate its usage:
```
Comment (agneszitte) [inline doc/controls/ShadowContainer.md:55]:
```suggestion
As mentioned `IsInner` is similar to `inset` of `box-shadow` in CSS. See below an image that illustrates its usage:
```

File: doc/controls/ShadowContainer.md [hunk 9]
```diff
@@ -3,7 +3,10 @@ uid: Toolkit.Controls.ShadowContainer
 ---
 
 ## Summary
-The `ShadowContainer` provides the possibility to add many-colored shadows to its content.
+The `ShadowContainer` provides the ability to add many-colored shadows to its content.
```
Comment (Xiaoy312) [inline doc/controls/ShadowContainer.md:6]:
```suggestion
The `ShadowContainer` provides the ability to add layer multiple shadows to its content.
```

File: doc/controls/ShadowContainer.md [hunk 10]
```diff
@@ -3,7 +3,10 @@ uid: Toolkit.Controls.ShadowContainer
...
+
+## Package Reference
+A reference to [Uno.Toolkit.Skia.WinUI](https://www.nuget.org/packages/Uno.Toolkit.Skia.WinUI) package must be added to your Uno app so you can access `ShadowContainer` class and features.
```
Comment (Xiaoy312) [inline doc/controls/ShadowContainer.md:9]:
```suggestion
A reference to [Uno.Toolkit.Skia.WinUI](https://www.nuget.org/packages/Uno.Toolkit.Skia.WinUI) package must be added to your Uno app so you can access `ShadowContainer` class and its features.
```

File: doc/controls/ShadowContainer.md [hunk 11]
```diff
@@ -3,7 +3,10 @@ uid: Toolkit.Controls.ShadowContainer
...
 
 ## Remarks
 For now, the control simply adapts its corner radius to the content's corner radius. More complicated shapes, such as text or pictures with alpha, are not supported.
```
Comment (Xiaoy312) [inline doc/controls/ShadowContainer.md:12]:
```suggestion
Currently, the control simply mimic the shape of its content by size and by corner radius. Complicated shapes, such as text or pictures with alpha, are not yet supported.
```

File: doc/controls/ShadowContainer.md (hunk 8 above)
Comment (Xiaoy312) [reply doc/controls/ShadowContainer.md:55]:
```suggestion
As mentioned `IsInner` is similar to `inset` of `box-shadow` in CSS. See the images below that illustrate its effects:
```

File: doc/controls/ShadowContainer.md [hunk 12]
```diff
@@ -244,51 +265,120 @@ xmlns:utu="using:Uno.Toolkit.UI"
...
 </StackPanel>
 ```
 
 ![neumorphism built with 2 shadows](../assets/shadows-neumorphism.png)
+
+## Theme Switching support
+
+To ensure that the Shadows feature seamlessly integrates with [Theme Switching](xref:Uno.Features.Themes), it's essential to include each shadow definition within the Theme Dictionary. By doing so, you'll enable consistent shadow rendering across different themes.
+
+Here's a snippet demonstrating how to structure your Theme Dictionary for shadow support:
```
Comment (Xiaoy312) [inline doc/controls/ShadowContainer.md:321]:
```suggestion
To ensure that the Shadows feature seamlessly integrates with [Theme Switching](xref:Uno.Features.Themes), it's essential to include the shadow collection within the theme dictionary.

Here's a snippet demonstrating how to structure your theme dictionary for proper shadow support:
```

File: doc/controls/ShadowContainer.md [hunk 13]
```diff
@@ -244,51 +265,120 @@ xmlns:utu="using:Uno.Toolkit.UI"
...
+							Opacity="1" 
+							Spread="0" 
+							Color="{StaticResource ShadeShadowColor}" />
+			</utu:ShadowCollection>
+		</ResourceDictionary>
+	</ResourceDictionary.ThemeDictionaries>
+</ResourceDictionary>
+...
+
+<utu:ShadowContainer Shadows="{StaticResource ButtonShadows}">
```
Comment (Xiaoy312) [inline doc/controls/ShadowContainer.md:375]: IMPORTANT
```suggestion
<utu:ShadowContainer Shadows="{ThemeResource ButtonShadows}">
```

**Review by pictos — APPROVED** [review]

File: doc/controls/ShadowContainer.md [hunk 14]
```diff
@@ -3,10 +3,13 @@ uid: Toolkit.Controls.ShadowContainer
...
 ## Summary
-The `ShadowContainer` provides the possibility to add many-colored shadows to its content.
+The `ShadowContainer` provides the ability to layer multiple shadows to its content.
+
+## Package Reference
+A reference to [Uno.Toolkit.Skia.WinUI](https://www.nuget.org/packages/Uno.Toolkit.Skia.WinUI) package must be added to your Uno app so you can access `ShadowContainer` class and its features.
 
 ## Remarks
-For now, the control simply adapts its corner radius to the content's corner radius. More complicated shapes, such as text or pictures with alpha, are not supported.
+Currently, the control simply mimic the shape of its content by size and by corner radius. Complicated shapes, such as text or pictures with alpha, are not yet supported.
```
Comment (jeromelaban) [inline doc/controls/ShadowContainer.md:12]:
```suggestion
Currently, the control simply mimics the shape of its content by size and by corner radius. Complicated shapes, such as text or pictures with alpha, are not yet supported.
```

**Review by jeromelaban — APPROVED** [review]

**Review by agneszitte — APPROVED** [review]

**Comment by agneszitte** [comment]
@Mergify backport legacy/3x

**Pull request merged**